apache-maven/src/assembly/maven/conf/settings.xml

```diff
@@ -257,9 +286,11 @@ under the License.
   <!-- activeProfiles
    | List of profiles that are active for all builds.
    |
+  -->
   <activeProfiles>
-    <activeProfile>alwaysActiveProfile</activeProfile>
-    <activeProfile>anotherAlwaysActiveProfile</activeProfile>
+    <activeProfile>maven:core:central-repo</activeProfile>
```

Comment:
Where is used?

Reply:
I need to revert that part, that comes from the original commit.

Reply:
So this PR is based on #1139, please check only commit https://github.com/apache/maven/pull/1142/commits/1d7045dcf631926d3a6515bc8527d5e3241cc482